reference/catalog.v3.yml

```diff
@@ -439,6 +691,9 @@ paths:
         - `date_modified`
         - `calculated_price`
         - `base_variant_id`
+
+        **Usage Notes**
+        You can supply category IDs, but the channel ID array should remain empty.
```

Comment:
elsewhere we've changed this. do we want to keep the `channel ID array should remain empty` in this location?